index.js

```diff
@@ -186,6 +186,18 @@ app.on('ready', () => {
 				},
 			];
 
+			const darkModeToggle = {
+				label: 'Toggle Dark Mode',
```

Comment:
should be easy to add keybinding here

Reply:
Pushed a change just now that I think takes care of this. Is that keybinding acceptable?

Reply:
Pushed a change just now that I think takes care of this. Is that keybinding acceptable?

Reply:
Hmm, so just recompiled and it looks like `CommandorControl+Shift+L` accelerator is only working while the context menu is open. I think there's either an issue with this specific shortcut combo, or another corresponding change needs to be made elsewhere.

Reply:
sure thing. @seanoliver can take a look